fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/transactionprocessor/impl/ChangeOperation.java

```diff
@@ -108,7 +110,16 @@ private OffsetDateTime getCreatedDateTime() {
         } else if (loanCharge.isPresent() && loanCharge.get().getCreatedDate().isPresent()) {
             return loanCharge.get().getCreatedDate().get();
         } else if (loanTransaction.isPresent()) {
-            return loanTransaction.get().getCreatedDate().orElse(null);
+            if (loanTransaction.get().isInterestRefund()) {
```

Comment:
I dont think we need this magic here. Please undo.

Reply:
done